src/main/java/team4/footwithme/team/repository/CustomTeamRepositoryImpl.java

```diff
@@ -0,0 +1,41 @@
+package team4.footwithme.team.repository;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import lombok.RequiredArgsConstructor;
+import team4.footwithme.global.domain.IsDeleted;
+import team4.footwithme.member.domain.Gender;
+import team4.footwithme.member.domain.QMember;
+import team4.footwithme.team.domain.QTeamMember;
+import team4.footwithme.team.domain.TeamMember;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+public class CustomTeamRepositoryImpl implements CustomTeamRepository {
+
+    private final JPAQueryFactory queryFactory;
+
+    @Override
+    public Long countMaleByMemberId(List<TeamMember> teamMembers) {
+        QTeamMember teamMember = QTeamMember.teamMember;
+        QMember member  = QMember.member;
```

Comment:
이런식으로 안하셔도 아마 임포트가 될거에요
```suggestion
        import static team4.footwithme.team.domain.QTeamMember.teamMember;
        import static team4.footwithme.member.domain.QMember.member;
```
import 문에 이런식으로 작성해주셔도 되겠네요. 자동 임포트도 될거에요!

Reply:
아아 감사합니다!!